deeppavlov/core/layers/tf_layers.py

```diff
@@ -115,7 +115,7 @@ def dense_convolutional_network(units: tf.Tensor,
 
 
 def bi_rnn(units: tf.Tensor,
-           n_hidden: List,
+           n_hidden: Union[List, int],
```

Comment:
Are you sure it can be an `int`? This typing will contradict the doctstring.
Also, there's a `n_hidden_list[-1]` reference in the `Returns` block of the docstring

Reply:
According to tf docs parameter `n_hidden` of  `GRUCell` constructor is `int`. I fixed docstring mentions of `n_hidden`.